datasets/tsplib/a280.tsp

```diff
@@ -0,0 +1,287 @@
+NAME : a280
+COMMENT : drilling problem (Ludwig)
+TYPE : TSP
+DIMENSION: 280
+EDGE_WEIGHT_TYPE : EUC_2D
+NODE_COORD_SECTION
+1 288 149
```

Comment:
Seems like all these files are really graph/sparse matrices rather than general euclidian space positions. 
Are we supporting only positive integers within the same range? 
If yes this should be mentioned in the API doc. 
If not other test data should be added.

Reply:
We're supporting float data, the tsp225 file contains purely floats. There are no constraints on the range between the cities

Reply:
Gotcha. Should we test negative coordinates?

Reply:
Added [file](https://github.com/rapidsai/cugraph/pull/1360/files#diff-bf9f6511a73f6a7380290414afb8f3a2de6da4f155d3f0fa914e4ed1bf4f6cd1) with negative weights